src/services/infrastructure/storage-aggregator-resolver/storage.aggregator.resolver.service.ts

```diff
@@ -122,12 +113,16 @@ export class StorageAggregatorResolverService {
       return await this.getStorageAggregatorOrFail(space.storageAggregator.id);
     }
 
-    const query = `SELECT \`id\` FROM \`innovation_pack\`
+    const query = `SELECT \`account\`.\`storageAggregatorId\` FROM \`innovation_pack\`
```

Comment:
could you please add some comments, describing what and why this happens.
Why don't you use directly the aggregator of the account?